spec/presenters/features_spec.rb

```diff
@@ -7,7 +7,7 @@
   describe '.wiki_ed?' do
     context 'when the url is dashboard-testing.wikiedu.org' do
       before do
-        allow(ENV).to receive(:[]).with('dashboard_url').and_return('dashboard-testing.wikiedu.org')
+        allow(ENV).to receive(:[]).with('wiki_education').and_return('true')
```

Comment:
You can either change the 'context' description for these, or delete them altogether. After this change, they no longer test anything useful.